__tests__/app.test.js

```diff
@@ -191,12 +184,67 @@ describe('Testing for News app', () => {
       .get('/api/articles/99999')
       .expect(404)
       .then(({ body }) => {
-          expect(body.msg).toBe('Page not found');
+          expect(body.msg).toBe('Invalid Path');
       });
     });
    
   });
 
-});
 
-});
\ No newline at end of file
+  //Ticket 8
+  describe("8. GET /api/articles", () => {
+    test("200: check an article object has all property", () => {
+      return request(app)
+        .get(`/api/articles`)
+        .expect(200)
+        .then(({body:{ articles }}) => {
+          expect(articles).toHaveLength(12);
+          articles.forEach((article) => {
+            expect(article).toHaveProperty("created_at");
+            expect(article).toHaveProperty("title");
+            expect(article).toHaveProperty("body");
```

Comment:
Should not have the body key in article, make sure you don't send the body of the article back

Reply:
Fixed